### ISSUE TITLE
What is your lr scheduler? your comment does not match your code

### ISSUE THREAD

**Issue by RuiFeiHe** [miraclewkf/MobileNetV2-PyTorch#7]
# Decay LR by a factor of 0.1 every 7 epochs
    exp_lr_scheduler = lr_scheduler.StepLR(optimizer_ft, step_size=1, gamma=0.98)
